lib/Rex/CLI.pm

```diff
@@ -612,11 +612,20 @@ sub summarize {
   Rex::Logger::info( @failures . " out of " . @summary . " task(s) failed:",
     "error" );
 
-  Rex::Logger::info( "\t$_->{task} failed on $_->{server}", "error" )
-    foreach sort {
-         ncmp( $a->{task}, $b->{task} )
-      || ncmp( $a->{server}, $b->{server} )
-    } @failures;
+  foreach (
+    sort {
+           ncmp( $a->{task}, $b->{task} )
+        || ncmp( $a->{server}, $b->{server} )
+    } @failures
+    )
+  {
+    Rex::Logger::info( "\t$_->{task} failed on $_->{server}", "error" );
+    if ( $_->{error_message} ) {
+      for my $line ( split( $/, $_->{error_message} ) ) {
+        Rex::Logger::info( "\t\t$line", "error" );
```

Comment:
@kablamo 
Yes, this is has not much style ;-) but without it you won't see any error message.
For example, on one machine i hadn't installed IO::Pty. And when i tried to run rex i didn't saw the error message that it was missing the IO::Pty module. Because it tried to load this module in the fork and so it suppressed that error message.